### ISSUE TITLE
Add instructions on when it takes time to import

### ISSUE THREAD

**Issue by OlivierJM** [BongoHive/how-to-instructions#2]
It’s likely to happen that the starter code importing will take longer time than intended.
What a user should know 
- it will eventually finish the importing.
- click on the link shown in a left box as long as it shows green, it will take you to your newly created repository.

**Comment by OlivierJM** [comment]
There should be a section to let them know the above, another thing to tell them is that while it’s importing they can go to the next assignment and import it ahead of time like Cynthia suggested.

**Comment by alfeyo** [comment]
Truly. Do we add this separately in each of our assignment, or in this main one

**Comment by OlivierJM** [comment]
It’s mainly needed in this one as part of other instructions.
Send in a PR for review if you plan to work on this issue.

**Comment by alfeyo** [comment]
add a pr

**Comment by Joebbrien** [comment]
I have seen a the css introduction assignment has been linked to a repository, did this solve the issue with that was logged in by Big Zude?

**Comment by OlivierJM** [comment]
Yes it resolved it, the issue was caused by a wrong starter code link, classroom couldn’t find it.

**Comment by OlivierJM** [comment]
But we still need an instructions so that users know why to do in case the import takes longer.

**Comment by Joebbrien** [comment]
So I noticed but was not sure which link to link it to. What will they need to do in such a case, should they wait until the starter code is done importing?

**Comment by OlivierJM** [comment]
There are 2 cases,
It’s either there is an error like what was happening to the other one or it’s taking time. 
When everything is working fine, even if it takes time to finish it will still go ahead and create the repository for them, check my first comment above. 
So they can click the link to the newly created repo.

**Comment by Joebbrien** [comment]
Let me go through all the assignments to see that they are all linked to a repository. 

I don't think we will be able to control the delay when one is creating the starter code, they will just need to wait in this case bro.
Will update once am done with rechecking

**Comment by OlivierJM** [comment]
Yeah for the delay unless Gitbub is having issues, the classroom will successfully import the starter code even before the progress bar is done that’s what I meant above. I have tried this several time.

**Comment by Joebbrien** [comment]
Have done a run through all the assignments. They are all linked to existing repositories. We should not expect any errors as the one Zude got from now.

**Comment by OlivierJM** [comment]
@Joebbrien as much as we might not expect such an error again, it is better to be pro-active about it, it is bound to take time when importing.

**Comment by alfeyo** [comment]
Thank you, everyone, for bringing their expertise and experience around the table and
engaging in such fruitful, constructive and open exchanges throughout internship-program.
SPECIAL THANKS TO THE TEAM.
